scripts/reformat-shfmt

```diff
@@ -15,6 +15,6 @@ if ! which shfmt > /dev/null; then
 	exit 1
 fi
 
-for file in $(shfmt -f scripts tests/shell); do
-	shfmt -s -sr -w "$file"
-done
+shell_files=$(shfmt -f 'scripts' 'tests/shell')
+non_ignored=$(printf "%s\n" "$shell_files" | sed -nE 's/(.*)/'"'"'\1'"'"'/p' | xargs git check-ignore -vn | sed -ne 's/^::[ \t]*\(.*\)$/\1/p')
```

Comment:
Some versions of `sed` [do not support `\t`](https://stackoverflow.com/questions/2610115/why-is-sed-not-recognizing-t-as-a-tab). You can use the [POSIX character class `[:blank:]`](https://www.regular-expressions.info/posixbrackets.html) to solve this problem:

```sh
sed -E 's/^::[[:blank:]]*(.*)$/\1/'
```

.